Makefile

```diff
@@ -100,12 +100,10 @@ generate: ## Run go generate
 
 .PHONY: generate-crds
 generate-crds: ## Generate CRDs and Go types using kubebuilder
-	go run sigs.k8s.io/controller-tools/cmd/controller-gen crd paths=./apis/... output:crd:dir=deploy/helm-chart/crds
-	go run sigs.k8s.io/controller-tools/cmd/controller-gen object paths=./apis/...
+	go run sigs.k8s.io/controller-tools/cmd/controller-gen crd object paths=./apis/... output:crd:artifacts:config=config/crd/bases
```

Comment:
How come we have to specify the output path is still? I thought we were updating to use the default path

Reply:
It's to create a subdirectory inside the default path, so we can group all of our CRDs together and make it easier to do operations on them in the future. Like they do in https://github.com/kubernetes-sigs/gateway-api/tree/main/config/crd